Sources/Authenticator/States/AuthenticatorBaseState.swift

```diff
@@ -30,6 +30,12 @@ public class AuthenticatorBaseState: ObservableObject {
         self.credentials = credentials
     }
 
+    init(authenticatorState: AuthenticatorStateProtocol,
+         credentials: Credentials = Credentials()) {
```

Comment:
🤔 Could we maybe not default to `Credentials()` here in the base class init, but rather on the children's callsite?